ovirt-ansible-disaster-recovery.spec.in

```diff
@@ -37,6 +37,9 @@ sh build.sh install
 %license LICENSE
 
 %changelog
+* Thu Jan 10 2019 Eyal Shenitzky <eshenitz@redhat.com> - 1.2.0-1
+- Convert python scripts to support python3
```

Comment:
I'd rephrase '''Convert python scripts to support both Python 2 and Python 3'''

Reply:
Done